test/unit/pr_checker.test.js

```diff
@@ -161,8 +161,8 @@ describe('PRChecker', () => {
       const cli = new TestCLI();
 
       const expectedLogs = {
-        warn: [['49 hours left to land']],
-        info: [['This PR was created on Fri Oct 27 2017 (weekend in UTC)']]
+        warn: [['25 hours left to land']],
+        info: [['This PR was created on Fri Oct 27 2017 (weekday in UTC)']]
```

Comment:
Can you modify the `createdAt` date below so that the created day of the PR is a weekend? I believe 2017-10-28 should be a Saturday. Then we can change `now` below to 2017-10-29

Reply:
@joyeecheung is this what we need: commit #78796e6 ? Thank you!